draft-ietf-rtcweb-jsep.xml

```diff
@@ -1764,6 +1764,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     successfully, will be added to the current and/or
                     pending remote description according to the rules
                     defined for Trickle ICE.
+                    The ufrag is used to determine which
```

Comment:
A few things:
- since the ufrag is often per-mid, I think the ufrag check needs to come after the MID check on line 1771. 
- Not all candidates will have ufrags, so need to mention that ufrag is only checked if present
- It's not obvious when the ufrag is needed to resolve ambiguity, so a brief discussion of this case would be helpful